src/components/select/select.js

```diff
@@ -1673,4 +1673,13 @@ function SelectProvider($$interimElementProvider) {
     }
     return isScrollable;
   }
+
+}
+
+function shouldHandleKey(ev, $mdConstant) {
+  var char = String.fromCharCode(ev.keyCode);
+  var isNonUsefulKey = (ev.keyCode <= 31);
+
```

Comment:
We need a check to make sure the user isn't performing a keyboard combination. For example, we don't want to stop users from being able to close a window (Ctrl+W) or stop a Refresh (Ctrl+R) simply because one of the combination keys is valid. Basically, we should abort if Alt, Ctrl, or Meta is being held.

This isn't the same as the isMetaKey function, which is only checking for single key presses of meta keys.